src/vmq_queue_sup.erl

```diff
@@ -203,16 +161,16 @@ loop(State = #state{parent=Parent}, NrOfChildren) ->
 
 %% Kill all children and then exit. We unlink first to avoid
 %% getting a message for each child getting killed.
-terminate(#state{shutdown=brutal_kill}, _, Reason) ->
-    ets:delete_all_objects(?QUEUE_TAB),
+terminate(#state{shutdown=brutal_kill, queue_tab = QueueTab}, _, Reason) ->
+    ets:delete_all_objects(QueueTab),
     _ = [begin
              unlink(P),
              exit(P, kill)
          end || P <- get_keys(true)],
```

Comment:
Instead of also storing the pids in the Process Dictionary, we could rely only on ETS. In this case we'd need to ets:fold, unlink, exit, and then ets:delete_all_objects.

Should we implement this as part of this PR?